### ISSUE TITLE
Whats the state of this project ?

### ISSUE THREAD

**Issue by andreimc** [puranjayjain/youtube-dl-gui#187]
Are you still working on this ?

Regards,
Andrei

**Comment by puranjayjain** [comment]
I still have plans to continue developing this in my spare time. I also think it should be put on open collective or patreon so that it becomes self-sufficient. I'm also looking for co maintainers.

**Comment by star114** [comment]
I'm interested in your project 👍

**Comment by puranjayjain** [comment]
@star114 will you interested in co-maintainership of the project?

**Comment by star114** [comment]
@puranjayjain  yes, I wanna get a co-maintainership of this project!  I searched the project using youtube-dl having GUI, and found your project. I'm just using it and searching it now, but I cannot start running on my mac os.
By the way, I don't have any experiences with web-based applications, so I need to set up and follow the technics in your project code line.
If you are ok with that, I'm very happy to maintain this project with you together. 👍

**Comment by puranjayjain** [comment]
Sounds great

**Comment by star114** [comment]
@puranjayjain Do you have any plans to develop something? or should we arrange our development items?

**Comment by puranjayjain** [comment]
atm I'm a bit busy but I'll try to free up some time coming weekend. We could arrange some development items then

**Comment by star114** [comment]
Ok, it looks good.
